pinot-common/src/main/java/org/apache/pinot/common/config/TableNameBuilder.java

```diff
@@ -122,4 +122,11 @@ public static String extractRawTableName(@Nonnull String tableName) {
   public static boolean isTableResource(@Nonnull String resourceName) {
     return OFFLINE.tableHasTypeSuffix(resourceName) || REALTIME.tableHasTypeSuffix(resourceName);
   }
+
+  /**
+   * Return whether the given resource name represents an offline table resource.
+   */
+  public static boolean isOfflineTableResource(@Nonnull String resourceName) {
```

Comment:
Why not use getTableTypeFromTableName?

Reply:
Wanted to make one similar to the method above (isTableResource), but only for OFFLINE